messages/agent.publish.authoring-bundle.md

```diff
@@ -1,24 +1,28 @@
 # summary
 
-Publish an authoring bundle to your org, which results in a new or updated agent.
+Publish an authoring bundle to your org, which results in a new agent or a new version of an existing agent.
 
 # description
 
 An authoring bundle is a metadata type (named aiAuthoringBundle) that provides the blueprint for an agent. The metadata type contains two files: the standard metatada XML file and an Agent Script file (extension ".agent") that fully describes the agent using the Agent Script language.
 
-When you publish an authoring bundle to your org, a number of things happen. First, this command validates that the Agent Script file successfully compiles. If there are compilation errors, the command exits and you must fix the Agent Script file to continue. Once the Agent Script file compiles, then the authoring bundle metadata component is deployed to the org, and all associated agent metadata components, such as the Bot, BotVersion, and GenAiXXX components, are either created or updated. The org then either creates a new agent based on the deployed authoring bundle, or creates a new version of the agent if it already existed. Finally, all the new or changed metadata components associated with the new agent are retrieved back to your local DX project.
+When you publish an authoring bundle to your org, a number of things happen. First, this command validates that the Agent Script file successfully compiles. If there are compilation errors, the command exits and you must fix the Agent Script file to continue. Once the Agent Script file compiles, then the authoring bundle metadata component is deployed to the org. All associated agent metadata components, such as the Bot, BotVersion, and GenAiXXX components, are either created or updated in the org. Finally, all the new or changed metadata components associated with the new agent are retrieved back to your local DX project.
```

Comment:
This isn't quite accurate.  This is the publish flow:

1. validate the agent script (aka compile)
2. publish the agent script (it's actually agent JSON at this point but we can just refer to the agent script), which creates the Bot and GenAi* metadata or new versions of them.
3. retrieve the created or changed metadata (Bot, GenAi*)
4. deploy the AiAuthoringBundle